packs/feat-effects/effect-change-shape.json

```
@@ -0,0 +1,367 @@
+{
+    "_id": "Q8pD29NcGGuRxWwh",
+    "img": "systems/pf2e/icons/spells/beastmaster-trance.webp",
+    "name": "Effect: Change Shape",
```

```suggestion
    "name": "Effect: Change Shape (Werecreature)",
```
We shouldn't make it seem like there's one generic effect for all change shape